src/site/antora/antora.yml

```
@@ -38,10 +38,10 @@ version: ~
 start_page: index.adoc
 asciidoc:
   attributes:
-    project-github-url: "https://github.com/apache/logging-log4j-tools"
-    project-version: "1.2.3"
-    project-name: "Log4j Tools"
-    project-id: "log4j-tools"
+    project-github-url: "https://github.com/apache/logging-log4net"
+    project-version: "3.1.0"
```

You can leave this containing a value that it is clear a dummy one. This file is auto-generated anyway. Content here is just a stub.

```suggestion
    project-version: "1.2.3"
```

---

Thanks.